tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -311,6 +311,14 @@ _ when x.ToBoolean() is { } boolean => boolean,
                                       .WithKeys(ConfigurationKeys.ApmTracingEnabled)
                                       .AsBool(defaultValue: true);
 
+            var ciVisibilityEnabled = config
+                                     .WithKeys(ConfigurationKeys.CIVisibility.Enabled)
+                                     .AsBool(defaultValue: false);
+
+            // If CI Visibility is enabled then we should enable tracing.
+            _traceEnabled = _traceEnabled || ciVisibilityEnabled;
+            _apmTracingEnabled = _apmTracingEnabled || ciVisibilityEnabled;
```

Comment:
I think APMTracingEnabled should remaing as it is. The value should be handled elsewere (setting the CI sampler, for instance). Let's discuss this in slack

Reply:
reverted in [920e1fc](https://github.com/DataDog/dd-trace-dotnet/pull/6951/commits/920e1fc505fd28ee8f836f15c753956dfe30750e)